capactio/capact
# Remove unresolved fields of input and output TypeInstances from Engine APIs

## Description

Changes proposed in this pull request:

- Remove unresolved fields of input and output TypeInstances from Engine APIs:
     - name for output TypeInstance
     - typeRef for input TypeInstances

Unfortunately, it would be time-consuming to fill these fields properly as it would be a part of rendering process. As no one uses them, I think we can remove them safely.

## Testing

1. Run Capact from the PR (`make dev-cluster`)
2. Rebuild Capact CLI (`make build-tool-cli && mv ./bin/capact-darwin-amd64 /usr/local/bin/capact`)
2. Create Action with the [`typeinstance` greet example](https://github.com/capactio/hub-manifests/tree/main/examples/typeinstance)
2. Test Action query from GraphQL examples and observe updated API
